dashboard/lib/api/v1/school_autocomplete.rb

```diff
@@ -46,7 +46,7 @@ def self.search_by_zip?(query)
 
   # JSON serializer used by get_matches.
   class Serializer < ActiveModel::Serializer
-    attributes :nces_id, :name, :city, :state, :zip, :school_type, :latitude, :longitude
+    attributes :nces_id, :name, :city, :state, :zip, :school_type, :latitude, :longitude, :afe_high_needs
```

Comment:
This might not be the best -- this serializer is used in our school search dropdowns throughout the site, so adding this as an attribute would result in joins to `SchoolStatsByYear` to get school demographics every time.

Reply:
@code-dot-org/plc I have a bunch of tests failing b/c of this change, but before I fix, how bad of an idea is this in the first place? Do I need to come up with something else?

Reply:
I think ideally we would not be doing a join a lot of places for something we will only use in a specific scenario? But I don't have any background knowledge on this part of the code

Reply:
I feared this response :) I'll come up with something else.

Reply:
@molly-moen moved this into it's own controller action that isn't used anywhere else:

https://github.com/code-dot-org/code-dot-org/pull/35290/commits/c931c011a92f1b6e67054aaf866f051e95116f6c